aws/resource_aws_ssm_document.go

```diff
@@ -384,6 +420,31 @@ func resourceAwsSsmDocumentDelete(d *schema.ResourceData, meta interface{}) erro
 	return nil
 }
 
+func expandAttachments(a []interface{}) []*ssm.AttachmentsSource {
```

Comment:
Nit: Resources and their functions currently live in the shared `aws` Go package across all services and resources. We should prefer to name functions with their service until this shared package is split up.

```suggestion
func expandSsmAttachmentsSources(a []interface{}) []*ssm.AttachmentsSource {
```